src/Sulu/Bundle/PageBundle/Content/Structure/TestExtension.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\PageBundle\Content\Structure;
+
+use PHPCR\NodeInterface;
+use Sulu\Component\Content\Extension\AbstractExtension;
+
+class TestExtension extends AbstractExtension
```

Comment:
would not create a `TestExtension` class here instead use anonymous class: e.g.:

```php
private function createTestExtension(): AbstractExtension
{
     return new class extends AbstractExtension() {
         // ..
     };
}
```